Find a functionally equivalent code:android.view.ViewPropertyAnimator.cancel:COMMENT
Method Modifier: public      
Comment:/**
 * Cancels all property animations that are currently running or pending.
 */

@@ -7,11 +7,18 @@
         }
     }
     mPendingAnimations.clear();
+    mPendingSetupAction = null;
+    mPendingCleanupAction = null;
+    mPendingOnStartAction = null;
+    mPendingOnEndAction = null;
     mView.removeCallbacks(mAnimationStarter);
+    if (mRTBackend != null) {
+        mRTBackend.cancelAll();
+    }
 }

